pkg/transformers/json/transformer.go

```diff
@@ -14,8 +14,14 @@ import (
 
 const sep = "/"
 
-// ErrInvalidKey represents an invalid JSON key format.
-var ErrInvalidKey = errors.New("invalid object key")
+var (
+	// ErrInvalidKey represents an invalid JSON key format.
+	ErrInvalidKey = errors.New("invalid object key")
+
+	errUnknownFormat     = errors.New("unknown format of JSON message")
+	errInvalidFormat     = errors.New("invalid JSON object")
+	errInvalidNestedJSON = errors.New("invalid nested JSON object")
```

Comment:
capital letters?

Reply:
Fixed. All the specific errors are wrapped in the single `ErrTransform`.